lib/components/tabs/trigger/trigger.scss

```
@@ -0,0 +1,25 @@
+@import '@quill/color.semantic.scss';
+
+.tab-trigger {
+    &__icon-position {
+        &--left {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
```

```suggestion
            display: flex;
            align-items: center;
```
you don't have to specify `flex-direction: row` it's already the default behaviour.